### PR TITLE
Document attributes

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Uses attributes to document module capabilities.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
all modules

### CODE REVIEW

**Comment by felixfontein** [comment]
Rendered docs:
- https://ansible.fontein.de/collections/community/docker/current_container_facts_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_compose_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_config_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_container_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_container_exec_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_container_info_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_host_info_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_image_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_image_info_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_image_load_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_login_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_network_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_network_info_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_node_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_node_info_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_plugin_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_prune_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_secret_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_stack_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_stack_info_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_stack_task_info_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_swarm_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_swarm_info_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_swarm_service_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_swarm_service_info_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_volume_module.html#attributes
- https://ansible.fontein.de/collections/community/docker/docker_volume_info_module.html#attributes

File: plugins/doc_fragments/attributes.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    ACTIONGROUP_DOCKER = r'''
+options: {}
+attributes:
+    action_group:
+      description: Use C(group/docker) or C(group/community.docker.docker) in C(module_defaults) to set defaults for this module.
+      support: full
+      membership:
+        - community.docker.docker
+        - docker
```
Comment (briantist) [inline plugins/doc_fragments/attributes.py:45]: is this because of legacy pre-collection groups?

**Review by briantist — APPROVED** [review]

File: plugins/doc_fragments/attributes.py (hunk 1 above)
Comment (felixfontein) [reply plugins/doc_fragments/attributes.py:45]: Yes (the legacy short name redirects are defined here: https://github.com/ansible/ansible/blob/devel/lib/ansible/config/ansible_builtin_runtime.yml#L9679-L9742).

**Comment by felixfontein** [comment]
@briantist thanks!

**Pull request merged**